### ISSUE TITLE
!skip

### ISSUE THREAD

**Issue by mmattbtw** [mmattbtw/TwitchTunes#5]

**Comment by spofoh** [comment]
Im sorry if im wrong because i'm not that experienced in Python but this should be working or? FeelsDankMan:

    @commands.command(name="skip")
    async def skip_song_command(self, ctx):
        if ctx.author.is_mod or self.is_owner(ctx):
            sp.next_track(device_id=None)
            await ctx.send(f":) 🎶 Skipping song...")
        else:
            await ctx.send(f"@{ctx.author.name}, You dont have rights for this...")

**Comment by mmattbtw** [comment]
> Im sorry if im wrong because i'm not that experienced in Python but this should be working or? FeelsDankMan:
> 
> 
> 
>     @commands.command(name="skip")
> 
>     async def skip_song_command(self, ctx):
> 
>         if ctx.author.is_mod or self.is_owner(ctx):
> 
>             sp.next_track(device_id=None)
> 
>             await ctx.send(f":) 🎶 Skipping song...")
> 
>         else:
> 
>             await ctx.send(f"@{ctx.author.name}, You dont have rights for this...")

yeah there is code already in the bot, just commented out. if you want to make a pr with this go ahead
